ecommerce/configuration.rb

```diff
@@ -8,6 +8,7 @@
 require_relative "invoicing/lib/invoicing"
 require_relative "taxes/lib/taxes"
 require_relative "processes/lib/processes"
+require_relative "coupon_discounts/lib/coupon_discounts"
```

Comment:
it's interesting that you went with a separate bounded contexts - I thought it would live under the Pricing umbrella.

Reply:
I did went with that but was kinda unsure about it tbh. Was a good exercise on the architecture though